src/App.js

```diff
@@ -0,0 +1,21 @@
+import ProductCard from './ProductCard'
+
+function App({ data }) {
+  return (
+    <div>
+      {data.map(product => (
+        <ProductCard
+          product={product.name}
+          price={product.price}
+          description={product.description}
+          alcohol={product.alcohol}
+          mount={product.mount}
+          image={product.image}
+          key={product.name}
```

Comment:
Right now in ProductCard.js you only use image and price (and key). You dont need to give the other information towards ProductCard. 
That said, you can leave it if you know that you will use it later on.